src/main/java/lotto/step2/domain/LottoTicket.java

```diff
@@ -0,0 +1,56 @@
+package lotto.step2.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTicket {
+    private List<LottoNumber> lottoNumbers;
+
+    public LottoTicket(List<LottoNumber> lottoNumbers) {
+        Validation.isCorrectLottoCount(lottoNumbers);
+        Validation.isValidLotto(lottoNumbers);
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public int matchWinningNumber(LottoTicket winningNumber) {
+        int matchCount = 0;
+        for (LottoNumber num : lottoNumbers) {
+            matchCount += isContainNumber(winningNumber, num);
```

Comment:
```suggestion
            matchCount += winningNumber.contains(num);
```
과 같은 꼴로 리펙토링 해볼 수 있을까요?